### DISCUSSION TITLE
Peering Manager Presentation at LACNOG 2023

### DISCUSSION THREAD

**Discussion by tomasconstant** [peering-manager/peering-manager#731] [General]
Guillaume,

I would like to present how we, AS20473, are using peering manager at LACNIC 40 - LACNOG2023 and/or invite you to present it in the same forum. Shall we continue the discussion here?

Tomas

**Comment by gmazoyer** [comment]
Hey Tomas, feel free to talk about Peering Manager, it would be my pleasure and will be happy to assist in any way I can. If you want to take this matter privately, feel free to send me a private message on Slack or other social media.

**Reply by tomasconstant** [reply]
Thanks Guillaume for your answer. The event is in October but I'm going to work on a draft presentation next week. I will add a link here to it so you can comment, etc.

**Comment by gmazoyer** [comment]
For people wondering if they can watch the presentation, you can find it [here](https://www.youtube.com/watch?v=OrJn5JRZjz0). If you understand Spanish, give it a look.

Thanks Tomas sharing your experience.

**Reply by tomasconstant** [reply]
The presentation was translated to english live. This is the [translated presentation](https://www.youtube.com/watch?v=OroS7m6LFfE). Thank you Guillaume!
